test/multibase.spec.js

```diff
@@ -13,7 +31,7 @@ describe('multibase', () => {
 
     it('fails on no buf', () => {
       expect(() => {
-        multibase('base58')
+        multibase('base1')
```

Comment:
`base58` doesn't exist anymore amongst the constant.js names, I thought it'd be less clear to let it that way.
(like -are we actually testing that it fails on the absence of buffer or is it failing before because of the non existing name ? turns out to be the same functionally speaking, no matter what the first argument is it will fail for the good reason)